src/posts/interning-at-khan-academy.md

```diff
@@ -0,0 +1,55 @@
+title: "Interning at Khan Academy: from student to intern"
+published_on: November 19, 2016
+author: Shadaj Laddad
+team: Web Frontend
+...
+
+This past summer I had the amazing opportunity to work on a product I use as a student every day: Khan Academy. During my internship, I worked on the web frontend team on the amusingly-named X-on-Mobile initiative (XOM) to bring Khan Academy exercises to mobile devices. In this blog post, I’ll be going through a highly-compressed summary of my two months at Khan Academy.
+
+# Part 1: Ramping up
+
+My first day at Khan Academy was a whirlwind of activities. From setting up my laptop to having a 1:1 with my mentor and finally adding myself to the [interns page](https://www.khanacademy.org/about/our-interns), I had a lot of things to do. As soon as I settled down with my first deploy, I got to work on the XOM initiative, starting with improving the math keypad.
+
+As part of XOM, the team had designed a math keypad that could be used to type in expression responses to mathematics problems in a mobile-friendly way. I started by fixing some bugs, such as [disabling gestures](https://github.com/Khan/math-input/commit/1bf10fa807ea1d98a755e18b38d502c6c7d551b3) on the keypad's sidebar, which has navigation buttons that cannot be swiped around like the other keys. Once I had a good idea of how the keypad worked, I was able to start working on adding new features our designers had crafted such as using [square icons](https://github.com/Khan/math-input/commit/ac8f92b7dc708e4fb6d955b545e2891e62098ed1) for representing empty blocks. And, by working on these low-level details, I learned the difference between a good user experience and a *great* user experience: polish.
+
+At Khan Academy, we use React, something I had worked with before, but only in the context of [Scala.js](https://www.scala-js.org) applications. As part of making changes to the math keypad, I had to learn how to write idiomatic ES6 code. Working on Khan Academy gave me a great opportunity to use React with ES6 in a production environment, and left me with many ideas on how React.js could be used more seamlessly with Scala.js. For example, seeing how properties could be easily reused for different components in ES6 gave me ideas of different ways to model properties in Scala.js. In addition to learning React and ES6, I got a chance to work with the plethora of open-source libraries at Khan Academy. For example, I often used [Aphrodite](https://github.com/khan/aphrodite) to style React components, with which I learned how easy it is to use inline styles to bring your CSS styles and JavaScript code closer together.
```

Comment:
Split this paragraph starting at "In addition...", it's a new idea and should have a new paragraph.